R/reportEmi.R

```diff
@@ -539,8 +623,12 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
                setNames(dimSums(EmiFeCarrier[, , "build"], dim = 3) * GtC_2_MtCO2,
                           "Emi|CO2|Energy|Demand|+|Buildings (Mt CO2/yr)"),
                # industry emissions: fe carrier emissions - industry CCS
-               # (carbon captured by industry used for CCU is still accounted as industry emissions as only account negative emissions for the CO2 provider if carbon gets stored)
-               setNames((dimSums(EmiFeCarrier[, , "indst"], dim = 3) - dimSums(vm_emiIndCCS[, , emiInd37_fuel]* p_share_CCS, dim = 3) ) * GtC_2_MtCO2,
+               # (carbon captured by industry used for CCU is still accounted as industry emissions
+               # as only account negative emissions for the CO2 provider if carbon gets stored)
+               setNames((dimSums(EmiFeCarrier[, , "indst"], dim = 3)
+                         - dimSums(vm_emiIndCCS[, , emiInd37_fuel]* p_share_CCS, dim = 3)
+                         # substract synthetic and biogenic carbon contained in industrial feedstocks
+                         - dimSums(plastic_CDR, dim=3)) * GtC_2_MtCO2,
```

Comment:
Just a quick refresher for my memory. Here, we only substract the plastic carbon from biogenic and synthetic sources and not the carbon from fossil sources because fossil feedstock carbon is not accounted as emissions in ``EmiFeCarrier`` in the first place. Correct?

Reply:
Also this means that we account all (non-fossil) carbon stored in materials as negative emissions for industry.